src/main/java/sg/ncl/ConnectionProperties.java

```diff
@@ -429,4 +429,24 @@ public String getUsageStat(String teamId, String... paramString) {
         }
         return HTTP_MODE + sioAddress + ":" + sioPort + "/" + analyticsEndpoint + "/usage/teams/" + teamId + params.toString();
     }
+
+    //-------------------------------------
+    // RESERVATION
+    //-------------------------------------
+    public String getReservationStatus(String teamId) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations";
+    }
+
+    public String releaseNodes(String teamId, Integer numNodes) {
+        return HTTP_MODE + sioAddress + ":" + sioPort + "/" + teamEndpoint + "/" + teamId + "/reservations?numNodes=" + numNodes;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "/reservations?numNodes=" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)